quantmsio/core/feature.py

```
@@ -43,13 +50,18 @@ def extract_psm_msg(self, chunksize=2000000, protein_str=None):
         return map_dict, pep_dict
 
-    def transform_msstats_in(self, file_num=10, protein_str=None, duckdb_max_memory="16GB", duckdb_threads=4):
-        Msstats = MsstatsIN(self._msstats_in, self._sdrf_path, duckdb_max_memory, duckdb_threads)
-        for msstats in Msstats.generate_msstats_in(file_num, protein_str):
+    def transform_msstats_in(
+        self, file_num=10, protein_str=None, duckdb_max_memory="16GB", duckdb_threads=4
+    ):
+        msstats = MsstatsIN(
+            self._msstats_in, self._sdrf_path, duckdb_max_memory, duckdb_threads
+        )
+        for msstats in msstats.generate_msstats_in(file_num, protein_str):
```

Ruff (0.8.2)

Loop control variable `msstats` overrides iterable it iterates

(B020)